src/lambda/steps/2_readQueuedUrls.ts

```diff
@@ -30,13 +38,27 @@ export const readQueuedUrls = async (crawlContext: CrawlContext) => {
     batchUrlCount: totalBatchUrlCount,
   });
 
+  let queuedPaths = urlsToVisit.map((path) => ({
```

Comment:
Nit: Prefer `const` where you don't reassign variables :)

Reply:
Fixed